compiler/luci/import/src/Nodes/CircleSVDF.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Import/Nodes/CircleSVDF.h"
+
+#include <luci/IR/Nodes/CircleSVDF.h>
+
+#include <loco.h>
+
+#include <cassert>
+
+namespace luci
+{
+bool CircleSVDFBuilder::validate(const ValidateArgs &args) const
+{
+  const auto &inputs = args.op.inputs;
+
+  if (inputs.size() != args.op.inputs.size())
+    return false;
+
+  return true;
+}
+
+/**
+ * @brief  SVDF Node builder
+ *
+ * @note
+ */
+
+CircleNode *CircleSVDFBuilder::build(const circle::OperatorT &op,
+                                     GraphBuilderContext *context) const
+{
+  assert(context != nullptr);
+
+  auto graph = context->graph();
+  const std::vector<int32_t> &inputs = op.inputs;
+  const std::vector<int32_t> &outputs = op.outputs;
```

Comment:
(optional) Could you please use `auto` when possible? I suggest it because `auto` may help to reduce code changes in future if we need to refactor something (e.g. use `circle::Operator *` instead `circle::OperatorT &`).

Reply:
Okey, I will change it in separated PRs for this draft.